src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Packaging/ProjectTypeCapabilities.cs

```diff
@@ -26,7 +26,7 @@ internal class ProjectTypeCapabilities
                                       ProjectCapability.ProjectConfigurationsDeclaredDimensions + "; " +
                                       ProjectCapability.LanguageService + "; " +
                                       ProjectCapability.DotNet + "; " +
-                                      ProjectCapability.CachedProjectEvaluation;
```

Comment:
Are Shared Projects getting the same capability?

Reply:
Yes, that's the plan. Just adding more testing to bring shared projects into the fold.

Reply:
Have you verified F# projects work under this mode (you've opt'd them in)? They have a different way of displaying the tree so might be affected by this change.

Reply:
Not yet, F# is in the test matrix for the cache service checkin acceptance. So far only been using CPS, project-system, orchardcore and some synthetic test solutions of varying size for the tests.

How are F# different? So I know what to test for.

Reply:
Then specifically only opt in C# and VB: https://github.com/dotnet/project-system/blob/be0048c2e264608488391dee1a23921d47cb25fe/src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Packaging/ProjectTypeCapabilities.cs#L46-L57.

F# opt's into SortByDisplayOrder which requires evaluation to provide items in the order they appear in the project file.